### PR TITLE
Add cure53 review of node_exporter

### PR DESCRIPTION
**Pull request by RichiH**

### CODE REVIEW

File: content/docs/operating/security.md [hunk 1]
```diff
@@ -238,3 +238,7 @@ pre-built binaries provided by the project.
 
 For more details, please read the
 [final report of the audit](/assets/downloads/2018-06-11--cure53_security_audit.pdf).
+
+There was a
+[second audit by cure53](/assets/downloads/2020-07-21--cure53_security_audit_node_exporter.pdf)
+of node_exporter. This is in preparation for implementing TLS throughout the Prometheus ecosystem.
```
Comment (roidelapluie) [inline content/docs/operating/security.md:244]:
```suggestion
of Node Exporter. This is in preparation for implementing TLS throughout the Prometheus ecosystem.
```

Comment (roidelapluie) [reply content/docs/operating/security.md:244]: was / is ?

**Comment by RichiH** [comment]
DCO broken by that web commit, but I am fine merging as is.

**Comment by brian-brazil** [comment]
:+1:

**Pull request merged**